Core/GDCore/Project/Object.cpp

```diff
@@ -181,6 +185,10 @@ void Object::SerializeTo(SerializerElement& element) const {
   element.SetAttribute("tags", GetTags());
   objectVariables.SerializeTo(element.AddChild("variables"));
 
+  // Disabled while effects are not implemented at runtime.

```

Comment:
@HarsimranVirk I've disabled this for now, feel free to enable again as you work on the effects in the game engine/editor :)